graphql-mode.el

```diff
@@ -388,6 +389,59 @@ This is the function to be used for the hook `completion-at-point-functions'."
      (1 font-lock-variable-name-face)))
   "Font Lock keywords.")
 
+;;; Edit headers functionality:
+
+(defun graphql-edit-headers ()
+  "Edit graphql request headers interactively in a dedicated buffer.
+
+Open a buffer to edit `graphql-extra-headers'.  The contents of this
+buffer take precedence over the setting in `graphql-extra-headers'
+when sending a request."
+  (interactive)
+  (unless (equal major-mode 'graphql-mode)
+    (error "Not in graphql-mode, cannot edit headers"))
+  (let ((extra-headers-buffer
+         (concat "*Graphql Headers for " (buffer-name) "*")))
+    (pop-to-buffer extra-headers-buffer)
+    (if (and (string-empty-p (buffer-string)) graphql-extra-headers)
+        (progn
+          (insert (json-serialize graphql-extra-headers))
```

Comment:
Should we use `json-encode` as in other parts of the code in case native JSON is not available?

Reply:
Good idea, I'll see if I can replace it.